### PR TITLE
Duration to string fixes cli and tests

### PR DESCRIPTION
**Pull request by cbron**
depends on: https://github.com/rancher/norman/pull/315

This will fix: #577, #496

Also includes some fixes for rolloutController bugs

### CODE REVIEW

File: Dockerfile.dapper [hunk 1]
```diff
@@ -2,7 +2,7 @@ FROM golang:1.12.6-alpine3.9
 
 ARG DAPPER_HOST_ARCH
 ENV ARCH $DAPPER_HOST_ARCH
-ENV K3S_VERSION v0.9.1
+ENV K3S_VERSION v0.10.0
```
Comment (StrongMonkey) [inline Dockerfile.dapper]: don't bump it to v0.10.0, build will fail.

File: vendor/modules.txt [hunk 2]
```diff
@@ -467,7 +467,7 @@ github.com/rancher/mapper/builder
 github.com/rancher/mapper/definition
 github.com/rancher/mapper/values
 github.com/rancher/mapper/httperror
-# github.com/rancher/norman v0.0.0-20191015045353-cc004d32fcc9
+# github.com/rancher/norman v0.0.0-20191015045353-cc004d32fcc9 => /Users/cbron/apps/norman
```
Comment (cbron) [inline vendor/modules.txt]: WIP

File: vendor/github.com/rancher/norman/pkg/openapi/generate.go [hunk 3]
```diff
@@ -190,5 +190,8 @@ func typeAndSchema(typeName string, schemas *types.Schemas) (string, *types.Sche
 	if schema.InternalSchema != nil {
 		return "", schema.InternalSchema, nil
 	}
+	if typeName == "duration" && schema.PkgName == "k8s.io/apimachinery/pkg/apis/meta/v1" {
+		return "string", nil, nil
+	}
```
Comment (cbron) [inline vendor/github.com/rancher/norman/pkg/openapi/generate.go]: WIP: Need to vendor this in

File: go.mod [hunk 4]
```diff
@@ -13,6 +13,7 @@ replace (
 	github.com/jaguilar/vt100 => github.com/tonistiigi/vt100 v0.0.0-20190402012908-ad4c4a574305
 	github.com/jetstack/cert-manager v0.11.0 => github.com/rancher/cert-manager v0.5.1-0.20191021233300-3a070253aeda
 	github.com/linkerd/linkerd2 => github.com/StrongMonkey/linkerd2 v0.0.0-20191021165729-976fad67457a
+	github.com/rancher/norman => github.com/cbron/norman v0.0.0-20191030144811-a73102a76447
```
Comment (cbron) [inline go.mod]: WIP

**Pull request merged**